worker.js

```diff
@@ -66,61 +82,96 @@ app.post('/token', async (c) => {
 		'client_secret': config.clientSecret,
 		'redirect_uri': config.redirectURL,
 		'code': code,
-		'grant_type': 'authorization_code',
-		'scope': 'identify email'
+		'grant_type': 'authorization_code'
 	}).toString()
 
 	const r = await fetch('https://discord.com/api/oauth2/token', {
 		method: 'POST',
 		body: params,
 		headers: {
-			'Content-Type': 'application/x-www-form-urlencoded'
+			'Content-Type': 'application/x-www-form-urlencoded',
+			'User-Agent': 'DiscordBot (https://github.com/Erisa/discord-oidc-worker, 1.0.0)'
```

Comment:
I don't believe this is a valid user agent

Reply:
User-Agent: DiscordBot ($url, $versionNumber)

forgot the v but rest works